robotbase/src/main/java/com/team2357/lib/arduino/ArduinoSensor.java

```diff
@@ -0,0 +1,39 @@
+package com.team2357.lib.arduino;
+
+public class ArduinoSensor extends ArduinoUSBController{
+
+    private String m_deviceName;
+
+    public ArduinoSensor(String ttyDevice, String deviceName) {
+        super(ttyDevice);
```

Comment:
There is a bug where if the Arduino is not connected the robot code will crash via an IOException, this change should also find a way to fix that issue.